test/int_test.go

```diff
@@ -77,15 +77,26 @@ var tests = []*T{
 		"#DW jobdw type=xfs name=xfs-data-movement capacity=1TB",
 		"#DW copy_in source=/lus/global/test.in destination=$DW_JOB_xfs-data-movement/",    // TODO: Create a file "test.in" in the global lustre directory
 		"#DW copy_out source=$DW_JOB_xfs-data-movement/test.out destination=/lus/global/"). // TODO: Validate file "test.out" in the global lustre directory
-		WithPersistentLustre("xfs-data-movement-lustre-instance").                          // Manage a persistent Lustre instance as part of the test
+		WithPersistentLustre("xfs-data-movement-lustre-instance", "1TB").                   // Manage a persistent Lustre instance as part of the test
 		WithGlobalLustreFromPersistentLustre("/lus/global").
 		Serialized().
 		Pending(),
 
-	MakeTest("GFS2 with Containers (BLAKE)",
-		"#DW jobdw type=gfs2 name=gfs2-with-containers capacity=1TB",
-		"#DW container name=gfs2-with-containers profile=TODO DW_JOB_gfs2-with-containers",
-	).Pending(),
+	// Containers
+	// TODO: requires nnf-sos deployed with `feature/containers` branch
```

Comment:
This also requires containers are in the nnf-sos submodule. That's why the build is failing